### PR TITLE
Revert  sorting order

### PR DESCRIPTION
**Pull request by MrFlashAccount**
### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [ ] Unit tests have been written where possible.
- [ ] If meaningful changes were made to logic or tests affecting Enso Cloud integration in the libraries, 
      or the Snowflake database integration, a run of the [Extra Tests](https://github.com/enso-org/enso/actions/workflows/extra-nightly-tests.yml) has been scheduled.
  - If applicable, it is suggested to paste a link to a successful run of the Extra Tests.

### CODE REVIEW

**Review by somebody1234 — APPROVED** [review]
CR passed :white_check_mark:

File: app/common/src/services/Backend.ts [hunk 1]
```diff
@@ -1433,13 +1433,23 @@ export function compareAssets(a: AnyAsset, b: AnyAsset) {
 
   if (relativeTypeOrder !== 0) {
     return relativeTypeOrder
+  } else {
+    // We sort by modified date, because the running/recent projects should be at the top.
+    // but above the all folders
```
Comment (PabloBuchu) [inline app/common/src/services/Backend.ts]: Lets not start sentences with `but` and first word should be capitalised. Running projects should also appear first on the list but bellow catalogs

Comment (MrFlashAccount) [reply app/common/src/services/Backend.ts]: There should be comma instead, you're right

File: app/common/src/services/Backend.ts [hunk 2]
```diff
@@ -1433,13 +1433,23 @@ export function compareAssets(a: AnyAsset, b: AnyAsset) {
...
+    const aModified = Number(new Date(a.modifiedAt))
+    const bModified = Number(new Date(b.modifiedAt))
+    const modifiedDelta = aModified - bModified
+    if (modifiedDelta !== 0) {
+      // Sort by date descending, rather than ascending.
+      return -modifiedDelta
+    } else {
+      return (
+        a.title > b.title ? 1
```
Comment (PabloBuchu) [inline app/common/src/services/Backend.ts]: probably we need to lowercase the title before comparing them

File: app/gui/src/dashboard/services/__test__/Backend.test.ts [hunk 3]
```diff
@@ -0,0 +1,103 @@
+import { describe, expect, it } from 'vitest'
+import { Rfc3339DateTime } from '../../utilities/dateTime'
+import { AssetType, compareAssets, type AnyAsset } from '../Backend'
+
+describe('Backend', () => {
```
Comment (PabloBuchu) [inline app/gui/src/dashboard/services/__test__/Backend.test.ts:5]: lets mix capitalise letters, numbers and special characters in titles to catch more cases

**Comment by PabloBuchu** [comment]
@MrFlashAccount I know its revert so lets save the comments for feature PR

File: app/common/src/services/Backend.ts (hunk 2 above)
Comment (somebody1234) [reply app/common/src/services/Backend.ts]: we can use `localeCompare` for case insensitive compare

**Review by PabloBuchu — APPROVED** [review]

**Pull request merged**